packages/lesswrong/components/contents/SideItems.tsx

```diff
@@ -53,6 +52,13 @@ export const styles = (theme: ThemeType) => ({
   sidebar: {
     position: "relative",
     height: "100%",
+    width: RIGHT_COLUMN_WIDTH_WITH_SIDENOTES,
+    [sidenotesHiddenBreakpoint(theme)]: {
+      width: RIGHT_COLUMN_WIDTH_WITHOUT_SIDENOTES,
+      [theme.breakpoints.down('xs')]: {
+        width: RIGHT_COLUMN_WIDTH_XS,
+      }
+    },    
```

Comment:
not clear to me that this is desirable in all circumstances where side items might be present, do you know what motivated it?

Reply:
IIUC this will mess up column-widths on the post page if the post doesn't have sidenotes.